### PR TITLE
uploaded profile image and linked it to TEAM.md

### PR DESCRIPTION
**Pull request by EYRAM565**
Screenshot (24)

### CODE REVIEW

**Review by Akshay2996 — CHANGES_REQUESTED** [review]
Hi @EYRAM565 please refrain from updating the TEAM.md file because we are still discussing the possible approach for the image in the markdown.

You just upload the image to the teams folder only for now.

**Comment by EYRAM565** [comment]
okay noted. @Akshay2996 kindly check issues #417 for the instructions on uploading images, I just followed the steps.

**Comment by ni3-kdm** [comment]
> okay noted. @Akshay2996 kindly check issues #417 for the instructions on uploading images, I just followed the steps.

@EYRAM565 Please update your PR with upstream and follow the same instructions to add your image. New updates have different format and are better than previous one. If you are using VSCode then verify the `TEAM.md` in the preview mode.

**Comment by EYRAM565** [comment]
okay. should I remove the image from Team.md? @ni3-kdm

File: docs/TEAM.md [hunk 1]
```diff
@@ -6,7 +6,7 @@
 |[Adewale Abiola]( https://github.com/princehaybee )|I am a Nigerian. I have basic knowledge of HTML,CSS and Javascript.|
 |[Micheal Aderinto](https://github.com/ad3rinto)| Manchester UK, Hoping this will be my first step in the world of solving world problems with code.|
 |Usha Kiran ([K!r@nD0n](https://twitter.com/ushakiran_m)) | I am a student from India. I just started my OpenSource journey with TWINDLE. I am currently learning #JavaScript.|
-|Eyram ([Ey_kwasi](https://twitter.com/Ey_kwasi))| I am from Ghana and I have no experience, started learning web development some months ago. I am participating to learn and also make twindle a success.|
+|<img src="docs\images\team\eyram.jpg" >Eyram ([Ey_kwasi](https://twitter.com/Ey_kwasi))| I am from Ghana and I have no experience, started learning web development some months ago. I am participating to learn and also make twindle a success.|
```
Comment (ni3-kdm) [inline docs/TEAM.md:9]: @EYRAM565 Please match your format with [this](https://github.com/twindle-co/twindle/pull/451/files#diff-35d64f6afeed8fc6d1c2263211980cbb6c2ec53ae38a9e471881b31e50e8d8daR74).

Comment (EYRAM565) [reply docs/TEAM.md:9]: okay.

**Review by ni3-kdm — APPROVED** [review]
PR Looks fine!

**Review by pranavgoel29 — APPROVED** [review]

**Pull request merged**